x/clerk/side_handler.go

```diff
@@ -94,7 +94,8 @@ func SideHandleMsgEventRecord(
 		return hmCommon.ErrorSideTx(hmCommon.ErrInvalidMsg)
 	}
 
-	if !bytes.Equal(eventLog.ContractAddress.Bytes(), []byte(msg.ContractAddress)) {
+	// TODO - check this
```

Comment:
@murthyvitwit @jdkanani 
Failing test case because of upper case addr and lower case addr
Let me know how to resolve

Reply:
You need convert them to lower case and compare. Use String.lowercase method.

Reply:
Should we do it in the test case itself? The concern is we might make the test case work but would it cause any issues later.

Reply:
I remember we decided to store lowercase addresses to avoid such issues.

Reply:
@ssandeep yes, for now fix test case. As mentioned in chat, we need to review the code once to store them in lowercase. Let's take it in a separate PR. Also, let's create an issue for that.